### ISSUE TITLE
[BUG] n_eph >= 5 causes crash for symbolic regression

### ISSUE THREAD

**Issue by MilesCranmer** [darioizzo/dcgp#84]
**Describe the bug**
Large numbers of ephemeral constants causes Python to crash when doing symbolic regression.

**To Reproduce**
1. Copy the code from here: http://darioizzo.github.io/dcgp/notebooks/symbolic_regression_3.html
2. Choose `n_eph` to be 5 or larger in the argument to `dcgpy.symbolic_regression`.
3. Run the code.
This causes a Python crash.

**Screenshots**
Here is the log, with verbosity=1000 (I'm not sure what the highest verbosity level is)

```
2aab23143000-2aab23144000 ---p 00000000 00:00 0 
2aab23144000-2aab23384000 rw-p 00000000 00:00 0 
2aab24000000-2aab24021000 rw-p 00000000 00:00 0 
2aab24021000-2aab28000000 ---p 00000000 00:00 0 
2aab28000000-2aab28021000 rw-p 00000000 00:00 0 
2aab28021000-2aab2c000000 ---p 00000000 00:00 0 
555555554000-5555555af000 r--p 00000000 00:2c 4221074                    /mnt/home/mcranmer/miniconda3/envs/dcgpy/bin/python3.7
5555555af000-555555788000 r-xp 0005b000 00:2c 4221074                    /mnt/home/mcranmer/miniconda3/envs/dcgpy/bin/python3.7
555555788000-55555582f000 r--p 00234000 00:2c 4221074                    /mnt/home/mcranmer/miniconda3/envs/dcgpy/bin/python3.7
55555582f000-555555832000 r--p 002da000 00:2c 4221074                    /mnt/home/mcranmer/miniconda3/envs/dcgpy/bin/python3.7
555555832000-55555589b000 rw-p 002dd000 00:2c 4221074                    /mnt/home/mcranmer/miniconda3/envs/dcgpy/bin/python3.7
55555589b000-555559556000 rw-p 00000000 00:00 0                          [heap]
7ffffffdc000-7ffffffff000 rw-p 00000000 00:00 0                          [stack]
ffffffffff600000-ffffffffff601000 r-xp 00000000 00:00 0                  [vsyscall]

   Gen:        Fevals:     Best loss: Ndf size:  Compl.:
      0              0        1.41375         4         7
[I 22:01:59.528 NotebookApp] KernelRestarter: restarting kernel (1/5), keep random ports
```

**Environment (please complete the following information):**
 - OS: CentOS Linux release 7.8.2003 (Core)
 - Installation method: `conda create --name dcgpy --channel conda-forge python=3.7 numpy scipy matplotlib cython h5py pyfftw notebook tqdm jupyter jupyterlab mpi4py openmpi astropy ipywidgets dcgp-python -y`

Versions:

``` 
python: 3.7.5
dcgpy: 1.5
pygmo: 2.15.0
```

**Comment by CoolRunning** [comment]
I can confirm this bug. Here is another piece of code that can be used to investigate:
```python
import dcgpy
import pygmo as pg
import numpy as np

X = np.random.random( (5795, 6) )
Y = np.random.random( (5795, 1) )
ss = dcgpy.kernel_set_double(["sum", "diff", "mul", "sin", "cos"])

udp = dcgpy.symbolic_regression(
    points = X, labels = Y, kernels=ss(),
    rows = 8,
    cols = 50,
    n_eph = 6,
    levels_back = 40,
    multi_objective=True)

prob = pg.problem(udp)

uda  = dcgpy.momes4cgp(gen = 250, max_mut = 4)
alg = pg.algorithm(uda)
alg.set_verbosity(10)

pop = pg.population(prob, 20)
pop = alg.evolve(pop)
```
setting `n_eph` to any value greater than 4 results in an interpreter crash.

**Comment by darioizzo** [comment]
Fixed in PR #89. Feel free to reopen if its not the case.
